src/RaisedButton/RaisedButton.spec.js

```diff
@@ -91,4 +91,16 @@ describe('<RaisedButton />', () => {
       );
     });
   });
+
+  it('inherits fontSize from theme', () => {
+    const wrapper = mountWithContext(
```

Comment:
Could we use the shallow rendering instead? I don't think that we need to mount the component to test this behavior.